package.json

```diff
@@ -91,33 +91,33 @@
     "react": "18.2.0",
     "react-apollo": "~3.1.5",
     "react-dom": "18.2.0",
-    "react-hook-form": "^6.11.0",
-    "react-native": "0.71.8",
+    "react-hook-form": "^7.12.2",
+    "react-native": "0.72.3",
     "react-native-app-intro-slider": "^4.0.4",
     "react-native-autocomplete-input": "^5.1.0",
     "react-native-calendars": "1.1293.0",
     "react-native-collapsible": "^1.6.0",
     "react-native-elements": "^3.4.3",
     "react-native-expo-image-cache": "4.1.0",
-    "react-native-gesture-handler": "~2.9.0",
-    "react-native-get-random-values": "~1.8.0",
+    "react-native-gesture-handler": "~2.12.0",
+    "react-native-get-random-values": "~1.9.0",
     "react-native-gifted-chat": "^2.0.1",
-    "react-native-maps": "1.3.2",
+    "react-native-maps": "1.7.1",
     "react-native-markdown-display": "^7.0.0-alpha.2",
     "react-native-modal-dropdown": "https://github.com/donni106/react-native-modal-dropdown#feature/search-options",
     "react-native-progress": "^5.0.0",
     "react-native-qrcode-svg": "^6.1.2",
-    "react-native-reanimated": "~2.14.4",
+    "react-native-reanimated": "~3.3.0",
```

Comment:
This looks like a major update. Have you read the changelog for this package? How much does it affect our code?

Reply:
same here: "Reanimated 3.x doesn't introduce any breaking changes between 2.x and 3.x in terms of the API. All the code you've written in Reanimated v2 API works in 3.x without any changes."